docs/src/main/asciidoc/logging.adoc

```diff
@@ -115,21 +115,24 @@ The fully qualified name of the class that calls the `Log` methods is used as a
 In this example, the logger name would be `com.example.MyService`.
 <3> Finally, all calls to `Log` methods are rewritten to regular JBoss Logging calls on the logger field during the application build.
 
-WARNING: Only use the `Log` API in application classes, not in external dependencies.
-`Log` method calls that are not processed by Quarkus at build time will throw an exception.
+[WARNING]
+====
+Use the `Log` API only in application classes, not in external dependencies.
+`Log` method calls not processed by Quarkus at build time will result in an exception.
 
-[[log-api-extension-warning]]
```

Comment:
You're dropping an anchor here. Pretty sure it's referenced from somewhere, possibly another website. Please leave the anchor, moving it if necessary?

Reply:
The anchor is a leftover after a level-4 heading. I removed and moved its text into one WARNING box. I searched, and it is not called by anything, but I will put it back; there is no harm in having it there.